src/main/java/com/notably/commons/path/RelativePath.java

```diff
@@ -125,6 +135,7 @@ public int compareTo(RelativePath path) {
 
     @Override
     public boolean equals(Object object) {
+        requireNonNull(object);
```

Comment:
This should be removed! Usually, `equals` still accepts `null` inputs. It'll just return false if that's the case!